docs_user/modules/proc_completing-prerequisites-for-migrating-ceph-rgw.adoc

```diff
@@ -26,7 +26,7 @@ You must complete the following prerequisites before you begin the {Ceph} RGW mi
     +------------------------+    +----------------+
 ----
 
-. Log in to `controller-0` and check the `pacemaker` status to help you
+. Log in to `controller-0` and check the Pacemaker status to help you
 identify the information that you need before you start the RGW migration.
```

Comment:
"to help you identify" or just to identify? 
Maybe:
"Log in to `controller-0` and check the Pacemaker status to identify important information for the RGW migration."